cmd/daemon/apis/v6_test.go

```diff
@@ -0,0 +1,229 @@
+package apis
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/justinbarrick/fluxcloud/pkg/utils"
+	"github.com/lunarway/release-manager/internal/log"
+	"github.com/stretchr/testify/assert"
+	fluxevent "github.com/weaveworks/flux/event"
+	"go.uber.org/zap/zapcore"
+)
+
+func TestHandleV6(t *testing.T) {
+	log.Init(&log.Configuration{
+		Level: log.Level{
+			Level: zapcore.DebugLevel,
+		},
+		Development: true,
+	})
+
+	fakeExporter := &FakeExporter{}
+
+	apiConfig := APIConfig{
+		Server:   http.NewServeMux(),
+		Exporter: fakeExporter,
+		Log:      log.With("type", "apiConfig"),
+	}
+
+	err := HandleV6(apiConfig)
+	if err != nil {
+		assert.Failf(t, "failed with error", "failed with error %s", err)
+		return
+	}
```

Comment:
You can use `assert.NoError()` for the same.
```suggestion
	if !assert.NoError(t, err, "unexpected error from HandleV6") {
		return
	}
```